apps/components-e2e/src/components/context-dialog/context-dialog.po.ts

```diff
@@ -39,3 +39,14 @@ export const getActiveElementAriaLabel = ClientFunction(() => {
   }
   return element.getAttribute('aria-label') || '';
 });
+
+export const isCloseButtonActiveElement = ClientFunction(() => {
```

Comment:
NIT: please add jsdoc what you are doing there because it is not that describing when you see this selector `('.dt-context-dialog-trigger')`